dll/win32/shell32/wine/shpolicy.c

```diff
@@ -749,27 +800,455 @@ static POLICYDATA sh32_policy_table[] =
     "NoSMMyDocs",
     SHELL_NO_POLICY
   },
+
+#ifdef __REACTOS__
+  {
+    REST_NOSMMYPICS,
+    strExplorer,
+    "NoSMMyPictures",
+    SHELL_NO_POLICY
+  },
+  {
+    REST_ALLOWBITBUCKDRIVES,
+    strExplorer,
+    "RecycleBinDrives",
+    SHELL_NO_POLICY
+  },
+#endif
+
 /* 0x4000050 - 0x4000060 */
+#if (WINE_FILEVERSION_MAJOR >= 6)
   {
     REST_NONLEGACYSHELLMODE,
     strExplorer,
     "NoneLegacyShellMode",
     SHELL_NO_POLICY
   },
+#endif
+
 #ifdef __REACTOS__
+
+  {
+    REST_NOCONTROLPANELBARRICADE,
+    strExplorer,
+    "NoControlPanelBarricade",
+    SHELL_NO_POLICY
+  },
+
+// NOTE: REST_NOSTARTPAGE never really existed.
+
+#if (WINE_FILEVERSION_MAJOR >= 6)
+  {
+    REST_NOAUTOTRAYNOTIFY,
+    strExplorer,
+    "NoAutoTrayNotify",
+    SHELL_NO_POLICY
+  },
+  {
+    REST_NOTASKGROUPING,
+    strExplorer,
+    "NoTaskGrouping",
```

Comment:
NOTE: @StartForKillerMC might want to know this value.

Reply:
As well as the other ones for the recycle bin ;)

Reply:
idk why there is TaskbarGlomming entry then, as is for the exact same purpose

Reply:
it's like a second check?

Reply:
Where is that `TaskbarGlomming` entry?

Reply:
Ah that explains things